restapi/stories.go

```diff
@@ -190,6 +189,10 @@ func (api api) CreateStoriesFormData(w http.ResponseWriter, r *http.Request) ren
 	if err := api.database.Create(&story).Error; err != nil {
 		return rest.ErrInternal(api.logger, err)
 	}
+
+	// We set the story to be public when it is initially uploaded
+	story.IsVisible = true
```

Comment:
I thought the story which is intially uploaded is not public

Reply:
word, yeah good catch

Reply:
We can remove that cause we have the gorm default.